### ISSUE TITLE
Bush not absorbing damage after getting shot

### ISSUE THREAD

**Issue by JLNFN** [kinsi55/FN_BugTracker#2]
**Current Fortnite version:** 8.4.1
**Affected plattforms:** PC;Console;Mobile

### Details

So a few updates ago the bush got buffed to absorb any kind of hit and then get removed from the player who wore the bush. This feature only worked for a few days and just recently I came across this bug in a match of mine. I shot a guy who wore a bush and instead of removing his bush by hitting him I damaged him. The bush of him didnt get removed nor did it absorb the damage I hit him for.

### [Steps to reproduce](#Reproducible)

It is reporducible by shooting an enemy who is wearing a bush. The bush shouldn't absorb the damage and wont get removed from the enemy either. I'm not sure wether this bug appears everytime but it appeared in one of my matches.

**Comment by kinsi55** [comment]
This isnt really the definition of Reproducible I'm looking for, what is rather meant by that is that you have steps under which circumstances the bug occurs 100% of the time, or at least on the majority

**Comment by JLNFN** [comment]
Yeah, like I said I dont know wether its 100% reproducible or not as I just came across this bug earlier today.
